applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes_fractional.h

```diff
@@ -0,0 +1,546 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Uxue Chasco
+//  Co-authors:      Ruben Zorrilla
+//
+
+#pragma once
+#define  KRATOS_TWO_FLUID_NAVIER_STOKES_FRACTIONAL
+
+// System includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/variables.h"
+#include "includes/serializer.h"
+#include "includes/cfd_variables.h"
+#include "custom_elements/fluid_element.h"
+#include "custom_utilities/fluid_element_utilities.h"
+#include "utilities/geometry_utilities.h"
+#include "modified_shape_functions/tetrahedra_3d_4_modified_shape_functions.h"
+#include "modified_shape_functions/triangle_2d_3_modified_shape_functions.h"
+
+namespace Kratos
+{
+
+/*The "TwoFluidNavierStokesFractional" element is an element based on the Variational Multiscale Stabilization technique (VMS)
+    * which is designed for the solution of a two fluids (air and a non-air one) problems.
+    *
+    * A distinctive feature of the element is the use of 4 LOCAL enrichment functions, which allows to model
+    * a discontinuity in both the pressure field and in its gradient.
+    * The enrichment functions are obtained by duplicating all of the degrees of freedom of the element.
+    * Since the enrichment is performed elementwise, a purely local static condensation step is performed,
+    * meaning that no extra degrees of freedom are added..
+    *
+    * Since a jump in the pressure can be considered, the element shall be able to habdle moderate changes of the viscosity
+    * between the two fluids to be considered
+    * The main difference between this element and the generic one "TwoFluidNavierStokes" is that the Navier-Stokes momentum conservation equation is solved in two problems using a fractional splitting approach. This element refers to the second problem.
+    */
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+    template <class TElementData>
```

Comment:
Remove the class indentation. Long ago we decided to avoid it in namespaces for the sake of readability.